trikot-viewmodels-declarative/compose/build.gradle.kts

```diff
@@ -35,6 +35,7 @@ dependencies {
 
     api("androidx.compose.foundation:foundation:${Versions.JETPACK_COMPOSE_RUNTIME}")
     api("androidx.compose.material:material:${Versions.JETPACK_COMPOSE_RUNTIME}")
+    api("androidx.compose.material3:material3:${Versions.JETPACK_COMPOSE_MATERIAL_3}")
```

Comment:
can we not use implementation here?

Reply:
It was already added in compose-flow as api so I added it in the same way in compose-publisher but it could probably changed in both yeah